cli/command/plugin/push.go

```diff
@@ -69,7 +69,7 @@ func runPush(dockerCli command.Cli, opts pushOptions) error {
 
 	if !opts.untrusted {
 		repoInfo.Class = "plugin"
-		return image.PushTrustedReference(dockerCli, repoInfo, named, authConfig, responseBody)
+		return image.PushTrustedReference(dockerCli, repoInfo, named, authConfig, dockerCli.ConfigFile().ParseTrustPinning(), responseBody)
```

Comment:
Wondering if it would be better to pass the pinningconfig through `opts` (add new properties for this to the `pushOptions` type)

Also note that we already pass `dockerCli` as argument, so this information _could_ already be obtained inside `PushTrustedReference()` without changing it signature.